### PR TITLE
chinese translation of README.md and introduction.md

### PR DESCRIPTION
**Pull request by captainroy-hy**
Initial Chinese translation of 

- application_configuration.md, 
- README.md , 
- introduction.md, 
- workload_types.md, 
- practical_considerations.md 

is done.

#261

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -74,3 +74,10 @@ One of the easiest ways to contribute is to participate in discussions. There ar
 | Twitter      | [@oam_dev](https://twitter.com/oam_dev) |
 
 [how-it-works]: assets/how-it-works.png
+
+## Translations
+
+
+* [简体中文](i18n/zh-Hans/README.md)
```
Comment (xiang90) [inline README.md]: what is the plan to keep the translation up with the english version?

Comment (wonderflow) [reply README.md]: It's my mistake, means nothing. I think the english version should be removed here.

File: i18n/zh-Hans/README.md [hunk 2]
```diff
@@ -0,0 +1,67 @@
+# 开放云原生应用模型
```
Comment (wonderflow) [inline i18n/zh-Hans/README.md]: 还是决定改为 `开放应用模型`

File: i18n/zh-Hans/README.md [hunk 3]
```diff
@@ -0,0 +1,67 @@
...
+
+开放云原生应用模型（OAM）是一个用于构造云原生应用的规范。
+
+_Open Application Model_ 聚焦于将开发与运维角色的关注点分离，秉持模块化、可拓展、可移植的设计原则，帮助用户在K8S等平台上构造并交付应用。
+
+本项目正处于快速迭代阶段，可能衍生出重大修改版本（Breaking Changes）。本项目对社区公众完全开放，如果您对社区贡献感兴趣，那么您可以从浏览项目的问题队列（Issue Queue）开始。我们正在**云原生应用建模领域**不断探索，期待您的积极参与！
```
Comment (wonderflow) [inline i18n/zh-Hans/README.md]:
```suggestion
本项目正处于快速迭代阶段，可能会出现不兼容的修改。本项目对社区公众完全开放，如果您对社区贡献感兴趣，那么您可以从浏览项目的问题队列（Issue Queue）开始。我们正在**云原生应用建模领域**不断探索，期待您的积极参与！
```

File: i18n/zh-Hans/README.md [hunk 4]
```diff
@@ -0,0 +1,67 @@
...
+### 社区贡献
+
+请参考 [社区贡献](../../CONTRIBUTING.md) 指南，了解如何参与本项目的社区工作。
+
+其中最简单的参与方式是参与讨论，多种参与途径如下：
+
+| 参与方式        | 链接  |
+|---------------------|---|
+| 社区邮件 | https://groups.google.com/forum/#!forum/oam-dev |
+| 社区会议时间 | [自2019年10月22日 周二 10:30AM（PST时间）起每个双周](https://calendar.google.com/calendar?cid=dDk5YThyNGIwOWJyYTJxajNlbWI0a2FvdGtAZ3JvdXAuY2FsZW5kYXIuZ29vZ2xlLmNvbQ)  |
```
Comment (wonderflow) [inline i18n/zh-Hans/README.md]: 中文翻译的社区时间可以用北京时间时区

File: i18n/zh-Hans/introduction.md [hunk 5]
```diff
@@ -0,0 +1,56 @@
+# 导读
+
+本文将通过叙述故事的方式帮助您快速理解OAM（开放应用模型）。
+
+通常一个应用交付的生命周期可以概括为以下3个主要过程：
```
Comment (wonderflow) [inline i18n/zh-Hans/introduction.md]:
```suggestion
接下来，我们通过一个故事案例来看一个应用交付的生命周期，这个案例包括三个主要过程：
```

File: i18n/zh-Hans/introduction.md [hunk 6]
```diff
@@ -0,0 +1,56 @@
...
+>译者注：所谓“一致性”，假如我们遵循OAM规范说明编写了一份应用描述（包含**组件**与**应用配置**），然后交付给不同的实现了OAM的平台去付诸实施，在实施部署时，各个平台都能 **“读懂”** 这份遵循OAM规范说明的应用描述，并能按照描述中的要求去部署运行应用。
+
+基础设施运维人员负责声明、安装并且维护底层平台的基础服务。举例来说，假如应用将被部署都一个云平台上并向外暴露服务，那基础设施运维人员将会选择特定的负载均衡技术（开发人员和应用运维人员则不需要关心这一点）。
+
+该过程如下图所示
+
+![alt](../../assets/platform_arch.png)
+
+
+## 致平台建设人员
```
Comment (wonderflow) [inline i18n/zh-Hans/introduction.md]:
```suggestion
## 对于构建平台的基础设施运维
```

**Comment by captainroy-hy** [comment]
Upload the initial translation of 6.application_configuration.md

**Comment by captainroy-hy** [comment]
Upload the initial translation of 7.workload_types.md & 8.practical_considerations.md

**Comment by wonderflow** [comment]
Please help give a new pull request to this branch [`i18n`](https://github.com/oam-dev/spec/tree/i18n), thanks!

**Comment by captainroy-hy** [comment]
> Please help give a new pull request to this branch [`i18n`](https://github.com/oam-dev/spec/tree/i18n), thanks!

got it.

**Pull request closed without merging**